package_manifest.yaml

```diff
@@ -1084,6 +1084,7 @@ plugin_nonscl_packages:
     mosquitto: {}
     puppet-foreman_scap_client: {}
     puppetlabs-stdlib: {}
+    yggdrasil-worker-forwarder: {}
```

Comment:
one thing that I realized: `yggdrasil` itself is in client, but it really needs to be in both, right?

Reply:
Truth.